mstEuclid.py

```diff
@@ -0,0 +1,118 @@
+import math
+from christofidesTSP import *
```

Comment:
Isn't this a circular dependency?

Reply:
mm yeah maybe a little bit. I'll slap the function into christofides and commit back

Reply:
Sounds good